deployments/systemd/packages/Makefile

```diff
@@ -45,24 +48,10 @@ tarball:
 		-v $$(pwd)/dist/$(@):/dist \
 		$(IMAGE):$(vVERSION)-$(@)
 
-ubuntu18.04:
```

Comment:
One concern I have here is that this will cause the packages we build to be incompatible with older debian-based systems. Note that these are not deployment images, but the images we use to build the binaries and package them.

This should be aligned with libnvidia-container or the nvidia-container-toolkit.

Reply:
@elezar thanks for working on the follow-up. Are there any pending items for this PR?

Reply:
I think my concerns are addressed by #42.